### ISSUE TITLE
HTML & WGAC validation issue

### ISSUE THREAD

**Issue by moanrose** [tinymce/tinymce#2984]
When running the evaluated DOM containing Tiny i get some validation issues.

Apparently they fall info two different categories:

Aria attributes such as aria-level="0" and role="presentation" - this can be easily fixed in HTML5 by appending data to the attributes: data-aria-level="0" and data-role="presentation" 

IE backward compatabilty issues such as frameborder="0", allowtransparency="true" and hidefocus="1"

Are there any reason why these validation issues cannot be fixed?

**Comment by spocke** [comment]
What validation tool are you using? The aria-level can be fixed the IE legacy stuff might be fixed when we drop IE 8 support need to investigate that.

**Comment by moanrose** [comment]
I am using TotalValidator & validator.w3.org

Kind regards

**Comment by moanrose** [comment]
After having looked further into the issue, found that my previous suggestions regarding the data- prefix was wrong, instead the `role="presentation"` should be substituted with `role="button"`
